apps/envited.ascs.digital/app/dashboard/assets/page.tsx

```diff
@@ -1,5 +1,10 @@
+import { getTokensByIssuerId } from '../../../common/serverActions'
 import { DashboardAssets } from '../../../modules/Assets'
 
 export default async function Index() {
-  return <DashboardAssets />
+  const tokens = await getTokensByIssuerId()
```

Comment:
I don't really like calling functions `byId` without actually providing an id. I assume its coming from the the session later, but reading the function name makes it look like something is missing.

Something like getUsersIssuersTokens() (can use some improvement) describes better what actually happens.